src/index.js

```diff
@@ -0,0 +1,59 @@
+import './index.css';
+import Book from './modules/form.js';
+
+const bookTableBody = document.getElementById('bookTableBody');
+const addButton = document.getElementById('addButton');
+const titleInput = document.getElementById('titleInput');
+const authorInput = document.getElementById('authorInput');
+
+const books = [];
+
+function renderBooks() {
+  bookTableBody.innerHTML = '';
+
+  for (const book of books) { // eslint-disable-line
+    const row = document.createElement('tr');
+    const titleCell = document.createElement('td');
```

Comment:
- [OPTIONAL] It is not good practice to use `eslint-disable` statements. `Eslint` helps us to write clean, readable, maintainable and industry-standard JavaScript codes. Using the `eslint-disable` statement defeats these purposes. Kindly remove it. 👍 😎

Reply:
Thank you, I'll do it!